### PR TITLE
Added LeftShift to raw_ops

### LINKED ISSUE #5413: LeftShift

### PR DESCRIPTION
**Pull request by KareemMAX**
Close #5413

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py [hunk 1]
```diff
@@ -1624,3 +1624,33 @@ def test_tensorflow_Cholesky(
...
+        num_arrays=2,
+        shared_dtype=True,
+        min_value=0,
+        max_value=8,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.LeftShift"
+    ),
+)
+def test_tensorflow_RightShift(
```
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py:1642]: Seems like a typo. Shouldn't the function be named `test_tensorflow_LeftShift` ?

**Comment by YushaArif99** [comment]
@KareemMAX there seem to be some merge conflicts caused by the changes you made to the `ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py` file. Could you kindly resolve the conflicts? Other than that, the changes seem good and should be ready to merge.

**Comment by KareemMAX** [comment]
@YushaArif99 I have resolved the conflicts now, also the typo you mentioned I fixed it.

Thanks ❤

**Comment by YushaArif99** [comment]
@KareemMAX thanks for making the changes! Everything seems fine so I will merge this PR. Thanks a lot for your contribution 😊

**Pull request merged**